extension/chrome/elements/compose-modules/compose-input-module.ts

```diff
@@ -78,11 +78,20 @@ export class ComposeInputModule extends ViewModule<ComposeView> {
     return this.view.sendBtnModule.popover.choices.richtext;
   };
 
+  public checkInputLengthBeforePasting = (textToPaste: string, targetInputField: HTMLElement, ev: ClipboardEvent) => {
+    const currentLength = targetInputField.innerText.length;
+    const limit = 50000;
+    if (textToPaste.length + currentLength > limit) {
```

Comment:
Do we want to gracefully handle the situation when the paste (partially) replaces the existing content? This will happen if there is an active selection at the moment of pasting.

Reply:
Not sure, sir, since we only want to test the input limit. But if the process in this pr affects the performance significantly I think we could consider that.

Reply:
We want to check the resulting element size, e.g. replacing a 40k text with another 40k text should trigger no error, right?
82782ace0c4e135b72b09ebda98e681b8f447e64 -- is it looking good?

Reply:
the changes from https://github.com/FlowCrypt/flowcrypt-browser/commit/82782ace0c4e135b72b09ebda98e681b8f447e64 look good to me. I see that I missed that scenario for subtracting content. thank you.